extensions/wikia/ArticleVideoMVP/styles/premium-mvp.scss

```diff
@@ -0,0 +1,135 @@
+#premium-mvp-article-header {
+	display: block;
+}
+
+#premium-mvp-video {
+	background-position: center center;
+	background-size: cover;
+	cursor: pointer;
+	margin-bottom: 11px;
+	padding-top: 56%;
+	position: relative;
+	width: 100%;
+
+	&:before {
+		background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0), #000000);
+		bottom: 0;
+		content: '';
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 50%;
+	}
+
+	.video-title {
+		bottom: 41px;
+		color: #fff;
+		font-size: 24px;
+		left: 38px;
+		position: absolute;
+	}
+
+	.video-time {
+		bottom: 73px;
+		color: #656e78;
+		font-size: 14px;
+		font-weight: bold;
+		left: 38px;
+		position: absolute;
+		text-transform: uppercase;
+	}
+
+	.video-play-button {
+		background-image: url("/extensions/wikia/ArticleVideoMVP/images/play-button-solid.svg"); /* $wgCdnStylePath */
+		background-repeat: no-repeat;
+		background-size: contain;
+		bottom: 15px;
+		height: 120px;
+		position: absolute;
+		right: 15px;
+		width: 120px;
+	}
+
+	.video-thumbnail {
+		.close {
+			content: url("/extensions/wikia/ArticleVideoMVP/images/close.svg"); /* $wgCdnStylePath */
+			display: none;
+			padding: 7px;
+			position: absolute;
+			top: 0;
+			left: 0;
+			z-index: 1;
+		}
+	}
+
+	&.collapsed {
+		.video-container {
+			bottom: 40px !important;
+			position: fixed;
+			right: 20px !important;
+			transition: right 0.4s, bottom 0.4s, width 0.4s;
+			width: 225px !important;
+		}
+
+		.video-thumbnail {
+			background-position: center center;
+			background-size: cover;
+			display: block;
+			height: 140px;
+
+			.close {
+				display: block;
+				z-index: 10001;
+			}
+		}
+
+		.video-details {
+			align-items: center;
+			background: #000;
+			display: flex;
+
+			.video-details-left {
+				flex: 1;
+				margin-left: 10px;
+			}
+
+			.video-time {
+				font-size: 10px;
+				position: static;
+			}
+
+			.video-title {
+				font-size: 12px;
+				position: static;
+			}
+
+			.video-play-button {
+				height: 60px;
+				margin: 0;
+				position: static;
+				width: 60px;
+			}
+		}
+
+		#ooyala-article-video {
+			bottom: auto;
+			height: 140px;
+		}
+
+	}
+
+}
+
+#ooyala-article-video {
+	bottom: 0;
+	left: 0;
+	position: absolute;
+	right: 0;
+	top: 0;
+
+	>div {
+		// that's because ooyala sets max z-index value when player is switched to fullscreen mode
+		// and it doesn't remove this style when user leaves this mode
+		z-index: 10000 !important;
```

Comment:
There are multiple features using `z-index`. Please make sure this one isn't too low: https://github.com/Wikia/app/search?utf8=%E2%9C%93&q=%24zTop